parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -324,6 +346,92 @@ public static Statistics toParquetStatistics(
     return stats;
   }
 
+  public org.apache.parquet.column.statistics.Statistics fromParquetStatistics(
+      String createdBy, Statistics statistics, PrimitiveType type) {
+    SortOrder expectedOrder = isSignedOrderOkay(type) ? SortOrder.SIGNED : sortOrder(type);
```

Comment:
This is a short-lived change and the order used to produce the min and max should be passed along with the stats in the next release. I don't think it makes sense to pass more type info, the actual order, and whether to override the order into the `shouldIgnoreStatistics` method (which is public) only to change it next release, so I'd like to keep this here to minimize the changes.